daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/Daml2ScriptTestRunner.scala

```diff
@@ -11,7 +11,7 @@ import java.nio.file.Paths
 class Daml2ScriptTestRunner extends DamlScriptTestRunner {
   self: Suite =>
 
-  val darPath = Paths.get(BazelRunfiles.rlocation("daml-script/test/script-test-v1.dar"))
+  val darPath = Paths.get(BazelRunfiles.rlocation("daml-script/test/script-test-v2.dar"))
```

Comment:
Is this expected to stay, even when the default version is 2.1? As in, does this need a comment like the other dar changes?

Reply:
Yes, we don't have a `script-test.dar`. I'm not sure it makes sense to have one as any major version change will require a new dar.